mixes/containers/mix.js

```diff
@@ -17,7 +17,7 @@ class MixContainer extends React.Component {
     if (!mix) { return null }
 
     console.log('mix', mix)
-    const title = isLoading
+    const $title = isLoading
```

Comment:
how do you normally name react element variables as opposed to normal things like strings?